packages/sources/dlc-btc-por/src/transport/proof-of-reserves.ts

```diff
@@ -0,0 +1,219 @@
+import { TransportDependencies } from '@chainlink/external-adapter-framework/transports'
+import { ResponseCache } from '@chainlink/external-adapter-framework/cache/response'
+import { AdapterResponse, makeLogger, sleep } from '@chainlink/external-adapter-framework/util'
+import { SubscriptionTransport } from '@chainlink/external-adapter-framework/transports/abstract/subscription'
+import { EndpointContext } from '@chainlink/external-adapter-framework/adapter'
+import { Requester } from '@chainlink/external-adapter-framework/util/requester'
+import { ethers } from 'ethers'
+import { hex } from '@scure/base'
+import { p2tr, p2tr_ns, taprootTweakPubkey, TAPROOT_UNSPENDABLE_KEY } from '@scure/btc-signer'
+import { BaseEndpointTypes } from '../endpoint/proof-of-reserves'
+import abi from '../config/dlc-manager-abi.json'
+import {
+  BitcoinNetwork,
+  BitcoinTransaction,
+  BitcoinTransactionVectorOutput,
+  getBitcoinNetwork,
+  RawVault,
+} from './utils'
+
+const logger = makeLogger('dlcBTC PoR')
+
+export type TransportTypes = BaseEndpointTypes
+
+export class DLCBTCPorTransport extends SubscriptionTransport<TransportTypes> {
+  name!: string
+  responseCache!: ResponseCache<TransportTypes>
```

Comment:
[low priority] do we need this line or is this taken care of in the parent class?

Reply:
Good point, we don't need that 👍.